ingest_openapi.yaml

```diff
@@ -1,11 +1,15 @@
 openapi: 3.0.0
+servers:
+  - url: https://CANDIG_URL/ingest
```

Comment:
Do we have an automated way to interpolate this variable? I'd rather leave it blank if we don't.